ui/src/app/user-info/components/user-info-overview/user-info-overview.tsx

```diff
@@ -1,39 +1,165 @@
+import {Form, FormApi, FormValue, Nested, Text} from 'react-form';
+import {RouteComponentProps} from 'react-router';
+import {DataLoader, ErrorNotification, Page, Spinner} from '../../../shared/components';
+import {AppContext} from '../../../shared/context';
 import {services} from '../../../shared/services';
 
 require('./user-info-overview.scss');
 
-export const UserInfoOverview = () => (
-    <Page title='User Info' toolbar={{breadcrumbs: [{title: 'User Info'}]}}>
-        <div className='user-info'>
-            <div className='argo-container'>
-                <div className='user-info-overview__panel white-box'>
-                    <DataLoader key='userInfo' load={() => services.users.get()}>
-                        {userInfo =>
-                            userInfo.loggedIn ? (
-                                <React.Fragment key='userInfoInner'>
-                                    <p key='username'>Username: {userInfo.username}</p>
-                                    <p key='iss'>Issuer: {userInfo.iss}</p>
-                                    {userInfo.groups && (
-                                        <React.Fragment key='userInfo4'>
-                                            <p>Groups:</p>
-                                            <ul>
-                                                {userInfo.groups.map(group => (
-                                                    <li key={group}>{group}</li>
-                                                ))}
-                                            </ul>
+export class UserInfoOverview extends React.Component<RouteComponentProps<any>, {connecting: boolean}> {
+    public static contextTypes = {
+        router: PropTypes.object,
+        apis: PropTypes.object,
+        history: PropTypes.object
+    };
+
+    private formApiPassword: FormApi;
+
+    constructor(props: RouteComponentProps<any>) {
+        super(props);
+        this.state = {connecting: false};
+    }
+
+    public render() {
+        return (
+            <Page
+                title='User Info'
+                toolbar={{
+                    breadcrumbs: [{title: 'User Info'}],
+                    actionMenu: {
+                        items: [
+                            {
+                                iconClassName: 'fa fa-lock',
+                                title: 'Update Password',
+                                action: () => (this.showChangePassword = true)
+                            }
+                        ]
+                    }
+                }}>
+                <DataLoader key='userInfo' load={() => services.users.get()}>
+                    {userInfo => (
+                        <div>
+                            <div className='user-info'>
+                                <div className='argo-container'>
+                                    <div className='user-info-overview__panel white-box'>
+                                        userInfo.loggedIn ? (
```

Comment:
<img width="1235" alt="Screen Shot 2021-02-11 at 11 48 22 AM" src="https://user-images.githubusercontent.com/6250584/107690500-0cf4e100-6c5f-11eb-910f-c6f0a0bc2449.png">

Could you fix this? Looks like you just need a `{` on this line and a matching `}` at the end of line 64.

Reply:
fixed - also fixed line 70 & 128 which were missing the same braces